tests/ui/drop/if-let-rescope-suggestions.rs

```diff
@@ -0,0 +1,25 @@
+//@ edition:2024
+//@ compile-flags: -Z validate-mir -Zunstable-options
+//@ run-rustfix
+
+#![deny(if_let_rescope)]
+#![feature(if_let_rescope)]
```

Comment:
Nit: can you flip these two:

```suggestion
#![feature(if_let_rescope)]
#![deny(if_let_rescope)]
```

Reply:
Applied